ci/launch_cluster.sh

```diff
@@ -38,16 +38,24 @@ echo "Using: ${TEMPLATE}"
 envsubst <<EOF > "$CONFIG_PATH"
 ---
 launch_config_version: 1
-template_url: $TEMPLATE
 deployment_name: $DEPLOYMENT_NAME
-provider: aws
+installer_url: https://downloads.dcos.io/dcos/testing/master/dcos_generate_config.sh
```

Comment:
Should this just be `https://downloads.dcos.io/dcos/$CHANNEL/dcos_generate_config.sh`?

Reply:
Yep, should be something like
```
https://downloads.mesosphere.com/dcos-enterprise/testing/pull/1739/dcos_generate_config.ee.sh
```
where `testing/pull/1739` is your `$CHANNEL` var.

Reply:
yep, done